manifests/plugin/certguard.pp

```diff
@@ -1,8 +1,14 @@
 # @summary Pulp Certguard plugin
 class pulpcore::plugin::certguard {
-  package { 'python3-subscription-manager-rhsm':
-    ensure => present,
+  # TODO: this breaks 3.15 - b79786c561ea56054c03fe83527cb79db84c295a should have addressed this
+  # The pulpcore_version fact is not always available - needs a better solution.
+  if fact('pulpcore_version') == '3.14' {
```

Comment:
This walling off will need to be reflected in the unit tests -- https://github.com/theforeman/puppet-pulpcore/blob/master/spec/classes/plugin_certguard_spec.rb#L9